integration/check_test.go

```diff
@@ -101,7 +101,7 @@ func (s *SkopeoSuite) TestCopyWithLocalAuth(c *check.C) {
 	assertSkopeoSucceeds(c, wanted, "login", "--tls-verify=false", "--username="+s.regV2WithAuth.username, "--password="+s.regV2WithAuth.password, s.regV2WithAuth.url)
 	// copy to private registry using local authentication
 	imageName := fmt.Sprintf("docker://%s/busybox:mine", s.regV2WithAuth.url)
-	assertSkopeoSucceeds(c, "", "copy", "--dest-tls-verify=false", "docker://docker.io/library/busybox:latest", imageName)
+	assertSkopeoSucceeds(c, "", "copy", "--dest-tls-verify=false", fmt.Sprintf("%s:latest", testFQIN), imageName)
```

Comment:
(Absolutely non-blocking: Just `testFQIN+":latest"`, in these cases where `%s` is at the start, might be a tiny bit more readable. Also elsewhere.)

Reply:
wait...I don't need to use `fmt.Sprintf()`?!?!?!

Reply:
Probably should have googled that first lol :tired_face: